setup.py

```diff
@@ -36,6 +36,7 @@ def run(self):
     ],
     "nd2": ["nd2[legacy]>=0.2.0"],
     "dv": ["mrc>=0.2.0"],
+    # "bfio": ["bfio>=2.2.1.dev1"],  # excluded for licensing reasons?
```

Comment:
The license on the repo says MIT but bfio ships with the bioformats jar... I think you will likely face the same legal discussion we had and probably need to change your package to GPL...

https://github.com/PolusAI/bfio/tree/master/bfio/jars
https://github.com/PolusAI/bfio/blob/master/setup.py#L27

(If I am forgetting how package_data works then I could be wrong but iirc package_data ships whatever you put in that lookup _with_ the package meaning you are now redistributing bioformats jar which means GPL)

Reply:
If all of my legal stepthrough is correct then yes, excluded for licensing reasons. If not and `bfio` is truly MIT then happy to include.

Reply:
Yeah, I suspected that might be the case. I was lurking in that discussion even though I never commented because I though it might apply to bfio.